test/EndToEnd/GivenPortableApps.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+using Xunit;
+
+namespace EndToEnd
+{
+    public class GivenPortableApps : TestBase
+    {
+        [Theory]
+        [ClassData(typeof(SupportedNetCoreAppVersions))]
+        public void ItDoesNotRollsForwardToTheLatestVersion(string minorVersion)
+        {
+            var _testInstance = TestAssets.Get("TestAppSimple")
+                .CreateInstance(identifier: minorVersion)
+                .WithSourceFiles();
+
+            string projectDirectory = _testInstance.Root.FullName;
+
+            string projectPath = Path.Combine(projectDirectory, "TestAppSimple.csproj");
+
+            var project = XDocument.Load(projectPath);
+            var ns = project.Root.Name.Namespace;
+
+            //  Update TargetFramework to the right version of .NET Core
+            project.Root.Element(ns + "PropertyGroup")
+                .Element(ns + "TargetFramework")
+                .Value = "netcoreapp" + minorVersion;
+
+            project.Save(projectPath);
+
+            //  Get the version rolled forward to
```

Comment:
It's not actually supposed to roll forward, so change this comment to "Get the resolved version of .NET Core" or similar

Reply:
changed to suggested